tests/test_cli.py

```diff
@@ -61,27 +55,49 @@ def test_indicator_help(indicator, indname):
 @pytest.mark.parametrize(
     "indicator,expected",
     [
-        ("atmos.tg_mean", 272.15),
-        ("atmos.daily_temperature_range_variability", 0.0),
-        ("atmos.heating_degree_days", 6588.0),
-        ("atmos.solid_precip_accumulation", 31622400.0),
+        ("tg_mean", 272.15),
+        ("dtrvar", 0.0),
+        ("heating_degree_days", 6588.0),
+        ("solidprcptot", 31622400.0),
     ],
 )
 def test_normal_computation(
     tasmin_series, tasmax_series, pr_series, tmp_path, indicator, expected
 ):
-    tasmin = tasmin_series(np.ones(366,) + 270.15, start="1/1/2000")
-    tasmax = tasmax_series(np.ones(366,) + 272.15, start="1/1/2000")
-    pr = pr_series(np.ones(366,), start="1/1/2000")
-    ds = xr.Dataset(data_vars={"tasmin": tasmin, "tasmax": tasmax, "pr": pr})
+    tasmin = tasmin_series(
+        np.ones(
+            366,
+        )
+        + 270.15,
+        start="1/1/2000",
```

Comment:
If you remove the trailing `,` after `start`, does it still explode?

Reply:
It do. Thanks!